### ISSUE TITLE
Baselines for forecasting: Validatation or Test Dataset

### ISSUE THREAD

**Issue by nitinsinghgit** [argoai/argoverse-api#68]
Are the results of the baselines mentioned in the paper on the validation dataset or the test dataset?
Also, what are the OTHERS object_type mentioned in the csv files? Are they just cars or can they be bicyclist/ pedestrian as well? Is it possible to get more information on them?

**Comment by James-Hays** [comment]
Those numbers are reported on the test set. We suspect the numbers would be pretty similar on the validation set (the distributions aren't too different), although that means less training data is available if you're setting aside the validation set for evaluation.

We'll have evaluation servers online very soon so that you can score on the test set, as well. 

With regard to your second question, are you asking about the forecasting dataset? The objects that aren't the "agent" or the "AV" can be bicycles and pedestrians. Many of the objects just beyond the map-defined driveable area but still in the ROI are pedestrians.

**Comment by nitinsinghgit** [comment]
Thanks @James-Hays. Waiting for the evaluation servers to compare the results on the test set. The results that I found with the LSTM model on the validation set were quite better than those mentioned in the paper so that got me confused.

second question: Yes, for the forecasting dataset. I think it would be helpful with Social Interaction Modelling to have information on each "OTHER" object type( whether its a car, bicyclist or a pedestrian).

**Comment by James-Hays** [comment]
You may have just beat our baseline :) 

We won't be releasing object class information for the objects in the forecasting dataset, but it might be worthwhile to try and estimate it on your own from the map. Velocity and position with respect to the lanes or sidewalk are a pretty big clue.

**Comment by johnwlambert** [comment]
@nitinsinghgit You can find the forecasting baselines here: https://github.com/jagjeet-singh/argoverse-forecasting
